server/documents/behaviors/api.html.eco

```diff
@@ -570,19 +570,19 @@ type        : 'UI Behavior'
               // test whether a JSON response is valid
               return response.success || false;
             },
-            onComplete: function(response) {
+            onComplete: function(response, element, xhr) {
               // always called after XHR complete
             },
-            onSuccess: function(response) {
+            onSuccess: function(response, element, xhr) {
               // valid response and response.success = true
             },
-            onFailure: function(response) {
+            onFailure: function(response, element) {
```

Comment:
Why only this do not receive xhr?

Reply:
Because the settings tab was missing it as well.... but i checked the api source and indeed the xhr parameter is also provided for the onFailure call 🙄 
... added now (here and also for settings tab) by fd5076cd41fd1ed884ab06948876b5a452d2dd18 🙂